### DISCUSSION TITLE
Where is the reference frame of the ego-vehicle set and how do you extract it for specifying sensor locations correctly?

### DISCUSSION THREAD

**Discussion by stevenbrills** [carla-simulator/carla#7126] [Q&A] [answered]
I have a real-world vehicle with a specific sensor configuration that has been described with respect to a known body-attached reference frame. I am trying to a sensor configuration within CARLA that matches the real world configuration but I can't seem to figure out where the reference frame of the vehicle is located. Is it at the center of the wheel-base and track-width? Where is this set and how can I extract relevant information to map it properly?

**Comment by noorikashif** [comment, marked as answer]
check out [this](https://carla.readthedocs.io/en/latest/coordinates/)

**Reply by stevenbrills** [reply]
Thanks, I'm glad they added it explicitly to the docs since verifying it manually was involved.
